src/consensus/consensus.h

```diff
@@ -31,7 +31,10 @@ static const int WITNESS_SCALE_FACTOR = 4;
 static const size_t MIN_TRANSACTION_WEIGHT = WITNESS_SCALE_FACTOR * 60; // 60 is the lower bound for the size of a valid serialized CTransaction
 static const size_t MIN_SERIALIZABLE_TRANSACTION_WEIGHT = WITNESS_SCALE_FACTOR * 10; // 10 is the lower bound for the size of a serialized CTransaction
 
-static bool fAssetsIsActive = false;
+#define UNUSED_VAR     __attribute__ ((unused))
```

Comment:
If UNUSED_VAR type will be used more than once then this will be nice to have however defining a new type to solve a single case does not seemed to be feasible. Although I liked the approach and happy to see that the problem solved at the source with bringing a use case, it shouldn't be encouraged to solve solitary cases with global definitions.